core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2881,6 +2881,52 @@ public String toString() {
       .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
       .setScope(Scope.WORKER)
       .build();
+  public static final PropertyKey WORKER_REVIEWER_BUFFER_SOFTLIMIT_BYTES =
+          new Builder(Name.WORKER_REVIEWER_BUFFER_SOFTLIMIT_BYTES)
+          .setDefaultValue("256MB")
+          .setDescription("This is used by the "
+              + "`alluxio.worker.block.reviewer.ProbabilisticBufferReviewer`. "
+              + "We attempt to leave a buffer in each storage directory. "
+              + "When the free space in a certain storage directory on the worker falls "
+              + "below this size, the chance that the Reviewer accepts new blocks "
+              + "into this directory goes down. "
+              + "This chance keeps falling linearly until it reaches 0, when the available "
+              + "space reaches the hard limit.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.WORKER)
+          .build();
+  public static final PropertyKey WORKER_REVIEWER_BUFFER_HARDLIMIT_BYTES =
+          new Builder(Name.WORKER_REVIEWER_BUFFER_HARDLIMIT_BYTES)
+          .setDefaultValue("64MB")
+          .setDescription("This is used by the "
+              + "`alluxio.worker.block.reviewer.ProbabilisticBufferReviewer`. "
+              + "When the free space in a storage dir falls below this size, "
+              + "the ProbabilisticBufferReviewer will stop accepting new blocks into it."
+              + "This is because we may load more data into existing blocks in the directory "
+              + "and their sizes may expand.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.WORKER)
+          .build();
+  public static final PropertyKey WORKER_REVIEWER_CLASS =
+      new Builder(Name.WORKER_REVIEWER_CLASS)
+          .setDefaultValue("alluxio.worker.block.reviewer.ProbabilisticBufferReviewer")
+          .setDescription("The strategy that a worker uses to review space allocation "
+              + "in the Allocator. Each time a block allocation decision is made by "
+              + "the Allocator, the Reviewer will review the decision and rejects it,"
+              + "if the allocation does not meet certain criteria of the Reviewer."
+              + "The Reviewer prevents the worker to make a bad block allocation decision."
+              + "Valid options include:"
+              + "`alluxio.worker.block.reviewer.ProbabilisticBufferReviewer`.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.WORKER)
+          .build();
+  public static final PropertyKey WORKER_REVIEWER_ENABLED =
```

Comment:
I had to add this property because many integration tests on tier storage depend on the deterministic behavior of the `Allocator`. Another option is to use a `NoopReviewer` that never rejects an allocation.

Reply:
I'm in favor of the pass through reviewer

Reply:
Added `AcceptingReviewer`